test/unit/fuzz/constructor/constuctor.tree

```diff
@@ -0,0 +1,19 @@
+constructor.t.sol
```

Comment:
The layout of this file is different from adjustDecimals.

Reply:
Forgot to convert this to a tree.